src/apis/AnonymousMedTechApi.ts

```diff
@@ -79,9 +86,12 @@ export class AnonymousMedTechApiBuilder {
       if (!this.authProcessId) {
         throw new Error("authProcessId is required");
       }
+      if (!this.authSpecId) {
+        throw new Error("authSpecId is required");
```

Comment:
You can add a unit test for this condition, in order to complete your test coverage (same for the condition to create in MedTechApi)